content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: dockershimからの移行
+weight: 10
+content_type: task
+---
+
+<!-- overview -->
+
+dockershimから他のコンテナランタイムに移行する際に知っておくべき情報を紹介します。
+
+Kubernetes 1.20で[dockershim deprecation](/blog/2020/12/08/kubernetes-1-20-release announcement/#dockershim-deprecation)が発表されてから、様々なワークロードやKubernetesインストールにどう影響するのかという質問が寄せられています。
```

Comment:
The link is failing.
![image](https://user-images.githubusercontent.com/42636694/148676067-21b3b747-6539-4e2a-b281-59522e5664f9.png)

Reply:
fixed.